app/src/main/kotlin/com/wire/android/ui/calling/ongoing/fullscreen/FullScreenTile.kt

```diff
@@ -68,6 +79,21 @@ fun FullScreenTile(
                 onSelfUserVideoPreviewCreated = sharedCallingViewModel::setVideoPreview,
                 onClearSelfUserVideoPreview = sharedCallingViewModel::clearVideoPreview
             )
+            LaunchedEffect(Unit) {
+                delay(200)
```

Comment:
maybe extract this also to the companion object?

Reply:
It's not visible on screen launch that's why it is different